src/backtrack.cpp

```diff
@@ -90,7 +97,7 @@ bool Backtrack::run(Coloring* aColoring, Graph* aG1, Graph* aG2, int32_t aNumTre
 	Timer t;
 	t.start();
 	bool result = backtrack(numMatching + numTreeNode);
-	searchTime = t.end();
+	t.end();
```

Comment:
Shouldn't the returned value still be assigned to `searchTime` as before?

Reply:
Good Point!! @NamYehyun I am moving global variable into each class like searchTime using getSearchTime() method, global variable is not good to handle this library with multi-trheaded.

Reply:
I see. I agree `searchTime` shouldn't be a global variable. But, my original question was about the value of `searchTime`. Where is `searchTime` being updated?

Reply:
Hi, @NamYehyun, I fixed to code to set searchTime variable :) Thanks.

Reply:
Thanks to all your works, now our code seems to be better organized. 😃